### ISSUE TITLE
One-Click 4-Color Sync Between Default Global Color Palette and Color Picker

### ISSUE THREAD

**Issue by cavalierlife** [pojome/elementor#6187]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
It is frustrating and time-wasting to go through the process of changing the colors in the Default Color Palette, only to have to repeat the process with the Color Picker. 

**Describe the solution you'd like**
The Color Picker has 8 colors. I believe the default colors in the Color Picker should consist of Black, White, Dark Gray, Light Gray, and the four colors in the default color palette. 

Most importantly, if the user changes the default color palette, there should be a way to change the color picker to include those four colors from the palette with just a click. 

So, let's say I change my palette. When I click Apply, a popup comes up and asks if I would also like to use those same colors in the color picker? If I say yes, the color picker now consists of black, white, dark gray, light gray, and my four palette colors. And just to be safe, the current color picker is saved as a backup and placed into the More Palettes area. That way, if I really didn't want to change my picker colors, I can always revert back. 

Of course, the user can still change any of the default black/white/grays as well as any of the palette 4-colors within the color picker, individually as desired. But a 1-click sync between the color palette and the color pickup would be ideal. 

Basically, I see the color picker as consisting of two sets of 4 colors. The first set - the basic black/white/grays (which the user could change to anything) and the second set of 4 colors. Clicking the YES button on the popup would only ever sync the color palette colors with the second set of 4 colors in the picker. The first set of  colors would never be affected by the sync.

**Describe alternatives you've considered**
I don't know of any alternatives that would solve this.

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

Thanks for the feedback.
